src/analyser.cpp

```diff
@@ -34,6 +34,7 @@ limitations under the License.
 #include "analysermodel_p.h"
 #include "analyservariable_p.h"
 #include "anycellmlelement_p.h"
+#include "commonutils.h"
```

Comment:
Why has this header been added?

Reply:
As a result of some cleaning up (see 57289480).